src/routers/genericRouters/routerEdges.ts

```diff
@@ -1119,6 +1120,48 @@ export class RouterEdges extends RouterFilterBy {
     return await cursor.all()
   }
 
+  // Given id for A, and A --(edge)--> B, return B's and edge's
```

Comment:
Can you please added a real application we use for function `getTargetAndEdgeSet` and `getSourceAndEdgeSet`?